source/helpers/engines.ts

```diff
@@ -21,7 +21,7 @@ export const markdownEngine = markdown({
   .use(markdownDefList)
   .use(markdownItAttrs)
   .use(markdownAnchor, {
-    level: [2],
+    level: [2,3],
```

Comment:
Adding this change for links like this: https://sass-lang.com/documentation/syntax/structure#css-statements